CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 # Changelog
 
-## Unreleased
+<!-- ## Unreleased -->
+
+## [23.0.0] - 2018-08-02
 
 ### Fixed
 * Updated `typescript-eslint-parser` dependency to version 17.0.1 in order to support TypeScript 3.
```

Comment:
Why did we stop linking to PRs?

Why isn't there an entry for, `Removed default prettier configurations`?

Reply:
Oh, there's an entry for the prettier configs.  Github is just choosing not to show it :/

Reply:
Looks like we forgot to link PRs in those changelog entries. 

That entry is covered by the next line: 
> `plugin:shopify/prettier` and `plugin:shopify/typescript-prettier` now defer Prettier's config to project's `.prettierrc`.

But that additional context you suggested is definitely helpful to readers.